packages/core-ethereum/crates/core-ethereum-rpc/Cargo.toml

```diff
@@ -13,16 +13,19 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 async-trait = { workspace = true }
-async-std = { version = "1.12.0", features = ["attributes"] }
+async-std = { workspace = true, features = ["unstable"] }
+async-stream = "0.3.5"
 console_error_panic_hook = { version = "0.1.7", optional = true }
-ethers = { version = "2", default-features = false }
+ethers = { workspace = true }
```

Comment:
Why is ethers also needed in core-ethereum-api? Trait? If  so, please, consider reexporting from the rpc package.